docs/patrol/advanced.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: Patrol - advanced
+---
+
+# Advanced
+
+### Patrol section in `pubspec.yaml`
```

Comment:
sometimes pubspec.yaml is in bold and sometimes in a codespan. Probably worth unifying.